### ISSUE TITLE
reword the "save" option

### ISSUE THREAD

**Issue by grrrrr** [FreeTubeApp/FreeTube#122]
I'd like to suggest rewording the `save` option that is currently with the player as it feels a bit confusing.

`save` is more closer to `download` in meaning on computer.    I would like to suggest using `favorite` or `bookmark` instead as it is more equivalent to what it does.

**Comment by PrestonN** [comment]
I can understand that as being confusing, especially for when proper download support comes in the future.  I'll change it to `Favorites` for the next release.

**Comment by K06RA** [comment]
Fixed this issue.. You can close it when you see my push.
